src/utils.cpp

```diff
@@ -75,4 +75,17 @@ std::string now(
     return ss.str();
 }
 
+std::string double_to_string(
+        const double& d)
+{
+    std::string str = std::to_string(d);
+    str.erase(str.find_last_not_of('0') + 1, std::string::npos);
```

Comment:
I think we are sure about it, but:
Are we completly sure the format will always be ''###.###`??
In case there is no `.` the right zeros are pretty important.

Maybe a small unittest of this single function will be nice.

Reply:
Addressed in ac5fc1a and in 1e12986